VimeoNetworking/Sources/NSError+Extensions.swift

```diff
@@ -161,7 +161,7 @@ public extension NSError
         
         if let userInfo = userInfo
         {
-            augmentedInfo.append(userInfo)
+            augmentedInfo.append(userInfo as! Dictionary<String, Any>)
```

Comment:
Rather than force casting here, we might want to reconsider the function argument to be `[String: Any]` rather than `[AnyHashable: Any]`

Reply:
I *think* this is a side effect of AFNetworking? Or at least the version that we're still using underneath everything. I would love to fix this but I think if you follow the chain up it ends up not being  a quick one.

Reply:
@jasonhawkins I did a project-wide find and replace of `[AnyHashable: Any]` -> `[String: Any]`, and it worked out. 

I think this would probably be a good change to make since entities that we're using as `[AnyHashable: Any]` in this framework are either JSON or `NSError` user info. In both of these cases, the key has to be a String. It's the JSON spec, and even `NSError` types `userInfo` as `NSDictionary<NSErrorUserInfoKey, id>`, where `NSErrorUserInfoKey` is a typedef for `NSString`.